cineast-core/src/main/java/org/vitrivr/cineast/core/features/AverageColorSuperpixelFh04.java

```diff
@@ -0,0 +1,65 @@
+package org.vitrivr.cineast.core.features;
+
+
+import java.awt.image.BufferedImage;
+import java.io.IOException;
+import java.util.List;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.vitrivr.cineast.core.color.ColorConverter;
+import org.vitrivr.cineast.core.color.ReadableLabContainer;
+import org.vitrivr.cineast.core.config.CacheConfig;
+import org.vitrivr.cineast.core.config.ReadableQueryConfig;
+import org.vitrivr.cineast.core.data.ReadableFloatVector;
+import org.vitrivr.cineast.core.data.raw.CachedDataFactory;
+import org.vitrivr.cineast.core.data.raw.images.MultiImage;
+import org.vitrivr.cineast.core.data.score.ScoreElement;
+import org.vitrivr.cineast.core.data.segments.SegmentContainer;
+import org.vitrivr.cineast.core.features.abstracts.AbstractFeatureModule;
+import org.vitrivr.cineast.core.util.ColorUtils;
+
+public class AverageColorSuperpixelFh04 extends AbstractFeatureModule {
```

Comment:
What is even the purpose of performing a superpixel aggregation, if all pixels are averaged afterward? Is there any significant difference in the output?